web/public/js/init-master.js

```diff
@@ -317,7 +318,8 @@
           templateUrl: '/dojos/template/dojo-list-index',
```

Comment:
This page seems to be broken as a result. Doesn't seem to be used though:
https://www.google.ie/?gws_rd=cr&ei=hesIWKnFM8nUgAbasrLwCA#q=site:%22http:%2F%2Fzen.coderdojo.com%22+intext:%22dojo-list-index%22